src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksDtoToPaymentWorksVendorConversionServiceImpl.java

```diff
@@ -46,6 +46,7 @@ public PaymentWorksVendor createPaymentWorksVendorFromPaymentWorksNewVendorReque
         PaymentWorksVendor stgVendor = new PaymentWorksVendor();
         if (newVendorDetailExists(pmwNewVendorRequestDetailDTO)) {
             stgVendor.setPmwVendorRequestId(pmwNewVendorRequestDetailDTO.getId());
+            stgVendor.setSupplierUploadStatus(PaymentWorksConstants.SupplierUploadStatus.PENDING_PAAT);
             populateNewVendorRequestingCompanyAttributes(stgVendor, pmwNewVendorRequestDetailDTO);
```

Comment:
Same comment as above. IMHO, I do not think status should not be set here.

Reply:
Ok, I'll remove the status setup here.